### ISSUE TITLE
Transclusion in Admonition

### ISSUE THREAD

**Issue by DutchPete** [nothingislost/obsidian-codemirror-options#79]
I have an Admonition with some text. Below the text I have a transclusion of a paragraph that is somewhere lower in the same note. So, it looks like this:

` ```ad-note `
my text

![[#^id number]]

` ``` `

In CodeMirror Options I have enabled “Render embeds”, but in Edit mode the embed does not render; it does in Preview.

**Comment by nothingislost** [comment]
Do you have the Render Code Blocks and Render Admonitions settings enabled? If that doesn't help, try turning off Syntax Highlighting within the Admonitions plugin.

**Comment by DutchPete** [comment]
Yes, I have Render Code Blocks and Render Admonitions settings enabled, and I never turn on Syntax Highlighting within the Admonitions plugin.

Since I opened this issue I switched to Live Preview, and CodeMirror Options is not yet CM6 compatible, I believe. If that is so, then this issue can be closed because it would not work before CMO is CM6-compatible, which hopefully will happen soon.
